src/common.jl

```diff
@@ -41,16 +65,26 @@ function solve{uType,tType,isinplace,AlgType<:ODEjlAlgorithm,F}(prob::AbstractOD
                       initstep=dt,
                       points=points)
 
-    # Reshape the result if needed
+
+    if save_start
```

Comment:
Sorry I take it back. the `save_start` -> `start_idx` is actually clear. I just missed this part of the updated api.